packages/react-components/react-storybook-addon/src/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const ADDON_ID = 'storybook/fluentui-react-addon';
-export const THEME_ID = `${ADDON_ID}/theme` as const;
+export const ADDON_ID = 'storybook_fluentui-react-addon';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1226 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27840&commitId=2ffc0fa3dd6e13bdee5de4d027764e1e8b45db31&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 710 screenshots unchanged, 0 screenshots with different dimensions and 1226 screenshots with visible difference.


  <details><summary><b>unknown</b> 1226 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.disabled.normal.chromium.png|32459| Changed|
|Accordion Converged.expandIcon   Icon-  .normal.chromium.png|32353| Changed|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|26746| Changed|
|Accordion Converged.icon   Icon-  .normal.chromium.png|32587| Changed|
|Accordion Converged.size.normal.chromium.png|60038| Changed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|26883| Changed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|26732| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|26732| Changed|
|AvatarGroup Converged.basic - Dark Mode.default.chromium.png|28255| Changed|
|AvatarGroup Converged.basic - High Contrast.default.chromium.png|28255| Changed|
|AvatarGroup Converged.basic.default.chromium.png|28255| Changed|
|AvatarGroup Converged.layoutPie - Dark Mode.default.chromium.png|18357| Changed|
|AvatarGroup Converged.layoutPie - High Contrast.default.chromium.png|18357| Changed|
|AvatarGroup Converged.layoutPie - RTL.default.chromium.png|18357| Changed|
|AvatarGroup Converged.layoutPie-1 - Dark Mode.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie-1 - High Contrast.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie-1 - RTL.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie-1.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie-2 - Dark Mode.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie-2 - High Contrast.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie-2 - RTL.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie-2.default.chromium.png|2157| Changed|
|AvatarGroup Converged.layoutPie.default.chromium.png|18357| Changed|
|AvatarGroup Converged.layoutStack - Dark Mode.default.chromium.png|25847| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|25847| Changed|
|AvatarGroup Converged.layoutStack.default.chromium.png|25847| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.chromium.png|110929| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|107744| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.chromium.png|110929| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|107746| Changed|
|AvatarGroup Converged.overflowContent.chromium.png|110929| Changed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|107762| Changed|
|AvatarGroup Converged.overflowIndicator - Dark Mode.default.chromium.png|28255| Changed|
|AvatarGroup Converged.overflowIndicator - High Contrast.default.chromium.png|28255| Changed|
|AvatarGroup Converged.overflowIndicator.default.chromium.png|28255| Changed|
|Badge Converged.filled.chromium.png|360745| Changed|
|Badge Converged.ghost.chromium.png|370797| Changed|
|Badge Converged.outline.chromium.png|366409| Changed|
|Badge Converged.size- extra-large.chromium.png|948631| Changed|
|Badge Converged.size- extra-small.chromium.png|558832| Changed|
|Badge Converged.size- large.chromium.png|804782| Changed|
|Badge Converged.size- medium.chromium.png|732277| Changed|
|Badge Converged.size- small.chromium.png|661208| Changed|
|Badge Converged.size- tiny.chromium.png|355659| Changed|
|Badge Converged.tint.chromium.png|375275| Changed|
|Badge Converged.width constrained.chromium.png|75257| Changed|
|Button Converged.As an anchor.default.chromium.png|50694| Changed|
|Button Converged.As an anchor.hover.chromium.png|50356| Changed|
|Button Converged.As an anchor.pressed.chromium.png|50439| Changed|
|Button Converged.Circular and icon only.default.chromium.png|50882| Changed|
|Button Converged.Circular and icon only.hover.chromium.png|50817| Changed|
|Button Converged.Circular and icon only.pressed.chromium.png|50832| Changed|
|Button Converged.Circular.default.chromium.png|50706| Changed|
|Button Converged.Circular.hover.chromium.png|50394| Changed|
|Button Converged.Circular.pressed.chromium.png|50471| Changed|
|Button Converged.Default.default.chromium.png|50694| Changed|
|Button Converged.Default.hover.chromium.png|50356| Changed|
|Button Converged.Default.pressed.chromium.png|50439| Changed|
|Button Converged.Disabled.default.chromium.png|50792| Changed|
|Button Converged.Disabled.hover.chromium.png|50792| Changed|
|Button Converged.Disabled.pressed.chromium.png|50792| Changed|
|Button Converged.Icon only.default.chromium.png|50869| Changed|
|Button Converged.Icon only.hover.chromium.png|50780| Changed|
|Button Converged.Icon only.pressed.chromium.png|50800| Changed|
|Button Converged.Outline Disabled.default.chromium.png|50727| Changed|
|Button Converged.Outline Disabled.hover.chromium.png|50727| Changed|
|Button Converged.Outline Disabled.pressed.chromium.png|50727| Changed|
|Button Converged.Outline.default.chromium.png|50694| Changed|
|Button Converged.Outline.hover.chromium.png|50668| Changed|
|Button Converged.Outline.pressed.chromium.png|50658| Changed|
|Button Converged.Primary Disabled.default.chromium.png|50856| Changed|
|Button Converged.Primary Disabled.hover.chromium.png|50856| Changed|
|Button Converged.Primary Disabled.pressed.chromium.png|50856| Changed|
|Button Converged.Primary.default.chromium.png|48255| Changed|
|Button Converged.Primary.hover.chromium.png|48254| Changed|
|Button Converged.Primary.pressed.chromium.png|48133| Changed|
|Button Converged.Size large - with long text wrapping.chromium.png|129380| Changed|
|Button Converged.Size large.default.chromium.png|63262| Changed|
|Button Converged.Size large.hover.chromium.png|62670| Changed|
|Button Converged.Size large.pressed.chromium.png|62828| Changed|
|Button Converged.Size medium - with long text wrapping.chromium.png|81790| Changed|
|Button Converged.Size small - with long text wrapping.chromium.png|56533| Changed|
|Button Converged.Size small.default.chromium.png|37975| Changed|
|Button Converged.Size small.hover.chromium.png|37739| Changed|
|Button Converged.Size small.pressed.chromium.png|37794| Changed|
|Button Converged.Subtle Disabled.default.chromium.png|50785| Changed|
|Button Converged.Subtle Disabled.hover.chromium.png|50785| Changed|
|Button Converged.Subtle Disabled.pressed.chromium.png|50785| Changed|
|Button Converged.Subtle.default.chromium.png|50737| Changed|
|Button Converged.Subtle.hover.chromium.png|50517| Changed|
|Button Converged.Subtle.pressed.chromium.png|50392| Changed|
|Button Converged.Transparent Disabled.default.chromium.png|50785| Changed|
|Button Converged.Transparent Disabled.hover.chromium.png|50785| Changed|
|Button Converged.Transparent Disabled.pressed.chromium.png|50785| Changed|
|Button Converged.Transparent.default.chromium.png|50737| Changed|
|Button Converged.Transparent.hover.chromium.png|50672| Changed|
|Button Converged.Transparent.pressed.chromium.png|50666| Changed|
|Button Converged.With icon after content.default.chromium.png|50658| Changed|
|Button Converged.With icon after content.hover.chromium.png|50250| Changed|
|Button Converged.With icon after content.pressed.chromium.png|50352| Changed|
|Button Converged.With icon before content.default.chromium.png|50658| Changed|
|Button Converged.With icon before content.hover.chromium.png|50250| Changed|
|Button Converged.With icon before content.pressed.chromium.png|50352| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|38153| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|36429| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|41600| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|37120| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|35570| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|40742| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|40218| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|40255| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|40347| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|34538| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|36432| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|40743| Changed|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|37120| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|35570| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|40742| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|34332| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|38153| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|36429| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|41600| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|35174| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|37120| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|35570| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|40742| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|34332| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|40218| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|40255| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|40347| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|40255| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|34538| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|36432| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|40743| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|35174| Changed|
|Card Converged.CardHeader.normal.chromium.png|159556| Changed|
|Card Converged.appearance.normal.chromium.png|442570| Changed|
|Card Converged.card templates.normal.chromium.png|31266| Changed|
|Card Converged.orientation.chromium.png|556451| Changed|
|Card Converged.size.normal.chromium.png|243158| Changed|
|Checkbox Converged.checked.active.chromium.png|5885| Changed|
|Checkbox Converged.checked.hover.chromium.png|5885| Changed|
|Checkbox Converged.checked.rest.chromium.png|5885| Changed|
|Checkbox Converged.circular+checked.default.chromium.png|5885| Changed|
|Checkbox Converged.circular+mixed.default.chromium.png|5885| Changed|
|Checkbox Converged.circular.default.chromium.png|5885| Changed|
|Checkbox Converged.disabled+checked.default.chromium.png|5885| Changed|
|Checkbox Converged.disabled+mixed.default.chromium.png|5885| Changed|
|Checkbox Converged.disabled.active.chromium.png|5885| Changed|
|Checkbox Converged.disabled.hover.chromium.png|5885| Changed|
|Checkbox Converged.disabled.rest.chromium.png|5885| Changed|
|Checkbox Converged.label-before - RTL.default.chromium.png|5885| Changed|
|Checkbox Converged.label-before.default.chromium.png|5885| Changed|
|Checkbox Converged.label-wrapping - RTL.default.chromium.png|6888| Changed|
|Checkbox Converged.label-wrapping.default.chromium.png|6888| Changed|
|Checkbox Converged.large - RTL.default.chromium.png|5952| Changed|
|Checkbox Converged.large+checked.default.chromium.png|5952| Changed|
|Checkbox Converged.large+circular+checked.default.chromium.png|5952| Changed|
|Checkbox Converged.large+circular+mixed.default.chromium.png|5952| Changed|
|Checkbox Converged.large+circular.default.chromium.png|5952| Changed|
|Checkbox Converged.large+label-wrapping - RTL.default.chromium.png|6955| Changed|
|Checkbox Converged.large+label-wrapping.default.chromium.png|6955| Changed|
|Checkbox Converged.large+mixed.default.chromium.png|5952| Changed|
|Checkbox Converged.large.default.chromium.png|5952| Changed|
|Checkbox Converged.mixed.active.chromium.png|5885| Changed|
|Checkbox Converged.mixed.hover.chromium.png|5885| Changed|
|Checkbox Converged.mixed.rest.chromium.png|5885| Changed|
|Checkbox Converged.no-label.default.chromium.png|5885| Changed|
|Checkbox Converged.required+label-before.default.chromium.png|5885| Changed|
|Checkbox Converged.required.default.chromium.png|5885| Changed|
|Checkbox Converged.unchecked - RTL.active.chromium.png|5885| Changed|
|Checkbox Converged.unchecked - RTL.hover.chromium.png|5885| Changed|
|Checkbox Converged.unchecked - RTL.rest.chromium.png|5885| Changed|
|Checkbox Converged.unchecked.active.chromium.png|5885| Changed|
|Checkbox Converged.unchecked.hover.chromium.png|5885| Changed|
|Checkbox Converged.unchecked.rest.chromium.png|5885| Changed|
|Combobox Converged.Appearance- filled-darker.default.chromium.png|6219| Changed|
|Combobox Converged.Appearance- filled-darker.focused.chromium.png|6219| Changed|
|Combobox Converged.Appearance- filled-darker.hover.chromium.png|6219| Changed|
|Combobox Converged.Appearance- filled-lighter.default.chromium.png|6219| Changed|
|Combobox Converged.Appearance- filled-lighter.focused.chromium.png|6219| Changed|
|Combobox Converged.Appearance- filled-lighter.hover.chromium.png|6219| Changed|
|Combobox Converged.Appearance- outline (default).default.chromium.png|5885| Changed|
|Combobox Converged.Appearance- outline (default).focused.chromium.png|5885| Changed|
|Combobox Converged.Appearance- outline (default).hover.chromium.png|5885| Changed|
|Combobox Converged.Appearance- underline.default.chromium.png|5868| Changed|
|Combobox Converged.Appearance- underline.focused.chromium.png|5868| Changed|
|Combobox Converged.Appearance- underline.hover.chromium.png|5868| Changed|
|Combobox Converged.Disabled option.active option.chromium.png|7826| Changed|
|Combobox Converged.Disabled option.default.chromium.png|7826| Changed|
|Combobox Converged.Disabled option.hover.chromium.png|7826| Changed|
|Combobox Converged.Disabled with value.default.chromium.png|5885| Changed|
|Combobox Converged.Disabled with value.focused.chromium.png|5885| Changed|
|Combobox Converged.Disabled with value.hover.chromium.png|5885| Changed|
|Combobox Converged.Disabled.default.chromium.png|5885| Changed|
|Combobox Converged.Disabled.focused.chromium.png|5885| Changed|
|Combobox Converged.Disabled.hover.chromium.png|5885| Changed|
|Combobox Converged.Invalid- filled-darker.default.chromium.png|6219| Changed|
|Combobox Converged.Invalid- filled-darker.focused.chromium.png|6219| Changed|
|Combobox Converged.Invalid- filled-darker.hover.chromium.png|6219| Changed|
|Combobox Converged.Invalid- filled-lighter.default.chromium.png|6219| Changed|
|Combobox Converged.Invalid- filled-lighter.focused.chromium.png|6219| Changed|
|Combobox Converged.Invalid- filled-lighter.hover.chromium.png|6219| Changed|
|Combobox Converged.Invalid- outline.default.chromium.png|5885| Changed|
|Combobox Converged.Invalid- outline.focused.chromium.png|5885| Changed|
|Combobox Converged.Invalid- outline.hover.chromium.png|5885| Changed|
|Combobox Converged.Invalid- underline.default.chromium.png|5868| Changed|
|Combobox Converged.Invalid- underline.focused.chromium.png|5868| Changed|
|Combobox Converged.Invalid- underline.hover.chromium.png|5868| Changed|
|Combobox Converged.Open with grouped options.active option.chromium.png|10781| Changed|
|Combobox Converged.Open with grouped options.default.chromium.png|10781| Changed|
|Combobox Converged.Open with grouped options.hover.chromium.png|10781| Changed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|7826| Changed|
|Combobox Converged.Open with inlinePopup.default.chromium.png|7826| Changed|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|7826| Changed|
|Combobox Converged.Open.active option.chromium.png|7826| Changed|
|Combobox Converged.Open.default.chromium.png|7826| Changed|
|Combobox Converged.Open.hover.chromium.png|7826| Changed|
|Combobox Converged.Option with long content.active option.chromium.png|9803| Changed|
|Combobox Converged.Option with long content.default.chromium.png|9803| Changed|
|Combobox Converged.Option with long content.hover.chromium.png|9803| Changed|
|Combobox Converged.Size- large.default.chromium.png|6019| Changed|
|Combobox Converged.Size- large.focused.chromium.png|6019| Changed|
|Combobox Converged.Size- large.hover.chromium.png|6019| Changed|
|Combobox Converged.Size- small.default.chromium.png|5751| Changed|
|Combobox Converged.Size- small.focused.chromium.png|5751| Changed|
|Combobox Converged.Size- small.hover.chromium.png|5751| Changed|
|Combobox Converged.With multiselect selection.default.chromium.png|7826| Changed|
|Combobox Converged.With placeholder.default.chromium.png|5885| Changed|
|Combobox Converged.With placeholder.focused.chromium.png|5885| Changed|
|Combobox Converged.With placeholder.hover.chromium.png|5885| Changed|
|Combobox Converged.With selection.active option.chromium.png|7826| Changed|
|Combobox Converged.With selection.default.chromium.png|7826| Changed|
|Combobox Converged.With selection.hover.chromium.png|7826| Changed|
|Combobox Converged.With value.default.chromium.png|5885| Changed|
|Combobox Converged.With value.focused.chromium.png|5885| Changed|
|Combobox Converged.With value.hover.chromium.png|5885| Changed|
|CompoundButton Converged.Circular and icon only.default.chromium.png|85852| Changed|
|CompoundButton Converged.Circular and icon only.hover.chromium.png|85697| Changed|
|CompoundButton Converged.Circular and icon only.pressed.chromium.png|85739| Changed|
|CompoundButton Converged.Circular.default.chromium.png|101262| Changed|
|CompoundButton Converged.Circular.hover.chromium.png|100228| Changed|
|CompoundButton Converged.Circular.pressed.chromium.png|100527| Changed|
|CompoundButton Converged.Default.default.chromium.png|101233| Changed|
|CompoundButton Converged.Default.pressed.chromium.png|100421| Changed|
|CompoundButton Converged.Disabled.default.chromium.png|101685| Changed|
|CompoundButton Converged.Disabled.hover.chromium.png|101685| Changed|
|CompoundButton Converged.Disabled.pressed.chromium.png|101685| Changed|
|CompoundButton Converged.Icon only.default.chromium.png|85829| Changed|
|CompoundButton Converged.Icon only.hover.chromium.png|85607| Changed|
|CompoundButton Converged.Icon only.pressed.chromium.png|85665| Changed|
|CompoundButton Converged.Outline Disabled.default.chromium.png|101449| Changed|
|CompoundButton Converged.Outline Disabled.hover.chromium.png|101449| Changed|
|CompoundButton Converged.Outline Disabled.pressed.chromium.png|101449| Changed|
|CompoundButton Converged.Outline.default.chromium.png|101233| Changed|
|CompoundButton Converged.Outline.hover.chromium.png|101242| Changed|
|CompoundButton Converged.Outline.pressed.chromium.png|101224| Changed|
|CompoundButton Converged.Primary Disabled.default.chromium.png|101798| Changed|
|CompoundButton Converged.Primary Disabled.hover.chromium.png|101798| Changed|
|CompoundButton Converged.Primary Disabled.pressed.chromium.png|101798| Changed|
|CompoundButton Converged.Primary.default.chromium.png|92937| Changed|
|CompoundButton Converged.Primary.hover.chromium.png|92935| Changed|
|CompoundButton Converged.Primary.pressed.chromium.png|92511| Changed|
|CompoundButton Converged.Size large - with long text wrapping.chromium.png|189479| Changed|
|CompoundButton Converged.Size large.default.chromium.png|126464| Changed|
|CompoundButton Converged.Size large.hover.chromium.png|124367| Changed|
|CompoundButton Converged.Size large.pressed.chromium.png|124978| Changed|
|CompoundButton Converged.Size medium - with long text wrapping.chromium.png|164213| Changed|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|113321| Changed|
|CompoundButton Converged.Size small.default.chromium.png|94802| Changed|
|CompoundButton Converged.Size small.hover.chromium.png|93524| Changed|
|CompoundButton Converged.Size small.pressed.chromium.png|93890| Changed|
|CompoundButton Converged.Subtle Disabled.default.chromium.png|101551| Changed|
|CompoundButton Converged.Subtle Disabled.hover.chromium.png|101551| Changed|
|CompoundButton Converged.Subtle Disabled.pressed.chromium.png|101551| Changed|
|CompoundButton Converged.Subtle.default.chromium.png|101362| Changed|
|CompoundButton Converged.Subtle.hover.chromium.png|100481| Changed|
|CompoundButton Converged.Subtle.pressed.chromium.png|100042| Changed|
|CompoundButton Converged.Transparent Disabled.default.chromium.png|101551| Changed|
|CompoundButton Converged.Transparent Disabled.hover.chromium.png|101551| Changed|
|CompoundButton Converged.Transparent Disabled.pressed.chromium.png|101551| Changed|
|CompoundButton Converged.Transparent.default.chromium.png|101362| Changed|
|CompoundButton Converged.Transparent.hover.chromium.png|101260| Changed|
|CompoundButton Converged.Transparent.pressed.chromium.png|101257| Changed|
|CompoundButton Converged.With icon after content.default.chromium.png|113881| Changed|
|CompoundButton Converged.With icon after content.pressed.chromium.png|112716| Changed|
|CompoundButton Converged.With icon before content.default.chromium.png|113881| Changed|
|CompoundButton Converged.With icon before content.pressed.chromium.png|112716| Changed|
|CounterBadge Converged - colors.default.chromium.png|31289| Changed|
|DatePicker Compat.DateRange- day.default.chromium.png|14877| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|14870| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|14870| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|14877| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|14877| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|14877| Changed|
|DatePicker Compat.Required.default.chromium.png|15314| Changed|
|DatePicker Compat.Underlined and required.default.chromium.png|14880| Changed|
|DatePicker Compat.Underlined.default.chromium.png|14880| Changed|
|DatePicker Compat.With label.default.chromium.png|15314| Changed|
|DatePicker Compat.allowTextInput.default.chromium.png|14880| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|14870| Changed|
|DatePicker Compat.default - Dark Mode.rest.chromium.png|14880| Changed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|14870| Changed|
|DatePicker Compat.default - High Contrast.rest.chromium.png|14880| Changed|
|DatePicker Compat.default.opened.chromium.png|14877| Changed|
|DatePicker Compat.default.rest.chromium.png|14880| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|14870| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|14870| Changed|
|DatePicker Compat.marked dates.default.chromium.png|14877| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|14870| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|14870| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|14877| Changed|
|Dialog.Fluid Actions End.chromium.png|117999| Changed|
|Dialog.Fluid Actions Start.chromium.png|118025| Changed|
|Dialog.actions position start & position end.chromium.png|118420| Changed|
|Dialog.actions position start.chromium.png|118689| Changed|
|Dialog.alert.chromium.png|118683| Changed|
|Dialog.default.chromium.png|118683| Changed|
|Dialog.nested.chromium.png|106279| Changed|
|Dialog.no actions.chromium.png|105871| Changed|
|Dialog.no title & no actions.chromium.png|91990| Changed|
|Dialog.no title.chromium.png|104803| Changed|
|Dialog.non-modal.chromium.png|140644| Changed|
|Dialog.scroll long content.chromium.png|467207| Changed|
|Dialog.title custom action.chromium.png|107520| Changed|
|Divider Converged - Horizontal.Appearance brand - Dark Mode.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance brand - High Contrast.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance brand.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance strong - Dark Mode.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance strong - High Contrast.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance strong.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance subtle - Dark Mode.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance subtle - High Contrast.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance subtle.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.End Aligned - RTL.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.End Aligned.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Inset.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Start Aligned - RTL.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Start Aligned.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.with content - Dark Mode.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.with content - High Contrast.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.with content - RTL.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.with content.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.without content - RTL.default.chromium.png|5367| Changed|
|Divider Converged - Horizontal.without content.default.chromium.png|5367| Changed|
|Divider Converged - Vertical.Center Aligned.default.chromium.png|2361| Changed|
|Divider Converged - Vertical.End Aligned.default.chromium.png|2361| Changed|
|Divider Converged - Vertical.Start Aligned.default.chromium.png|2361| Changed|
|Divider Converged - Vertical.inset.default.chromium.png|3424| Changed|
|Dropdown Converged.Appearance- filled-darker.default.chromium.png|6219| Changed|
|Dropdown Converged.Appearance- filled-lighter.default.chromium.png|6219| Changed|
|Dropdown Converged.Appearance- outline (default).default.chromium.png|5885| Changed|
|Dropdown Converged.Appearance- underline.default.chromium.png|5868| Changed|
|Dropdown Converged.Disabled option.active option.chromium.png|4014| Changed|
|Dropdown Converged.Disabled option.default.chromium.png|4014| Changed|
|Dropdown Converged.Disabled option.hover.chromium.png|4014| Changed|
|Dropdown Converged.Disabled with value.default.chromium.png|5885| Changed|
|Dropdown Converged.Disabled.default.chromium.png|5885| Changed|
|Dropdown Converged.Invalid- filled-darker.default.chromium.png|6219| Changed|
|Dropdown Converged.Invalid- filled-lighter.default.chromium.png|6219| Changed|
|Dropdown Converged.Invalid- outline.default.chromium.png|5885| Changed|
|Dropdown Converged.Invalid- underline.default.chromium.png|5868| Changed|
|Dropdown Converged.Open with grouped options.active option.chromium.png|4014| Changed|
|Dropdown Converged.Open with grouped options.default.chromium.png|4014| Changed|
|Dropdown Converged.Open with grouped options.hover.chromium.png|4014| Changed|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|4014| Changed|
|Dropdown Converged.Open with inlinePopup.default.chromium.png|4014| Changed|
|Dropdown Converged.Open with inlinePopup.hover.chromium.png|4014| Changed|
|Dropdown Converged.Open.active option.chromium.png|4014| Changed|
|Dropdown Converged.Open.default.chromium.png|4014| Changed|
|Dropdown Converged.Open.hover.chromium.png|4014| Changed|
|Dropdown Converged.Option with long content.active option.chromium.png|4014| Changed|
|Dropdown Converged.Option with long content.default.chromium.png|4014| Changed|
|Dropdown Converged.Option with long content.hover.chromium.png|4014| Changed|
|Dropdown Converged.Size- large.default.chromium.png|6019| Changed|
|Dropdown Converged.Size- small.default.chromium.png|5768| Changed|
|Dropdown Converged.With multiselect selection.default.chromium.png|4014| Changed|
|Dropdown Converged.With multiselect value.default.chromium.png|5885| Changed|
|Dropdown Converged.With placeholder.default.chromium.png|5885| Changed|
|Dropdown Converged.With selection.active option.chromium.png|4014| Changed|
|Dropdown Converged.With selection.default.chromium.png|4014| Changed|
|Dropdown Converged.With selection.hover.chromium.png|4014| Changed|
|Dropdown Converged.With value.default.chromium.png|5885| Changed|
|Field.Checkbox.default.chromium.png|8995| Changed|
|Field.Combobox.default.chromium.png|11670| Changed|
|Field.Dropdown.default.chromium.png|11670| Changed|
|Field.ProgressBar.default.chromium.png|10600| Changed|
|Field.RadioGroup.default.chromium.png|11569| Changed|
|Field.Select.default.chromium.png|11670| Changed|
|Field.Slider.default.chromium.png|9429| Changed|
|Field.SpinButton.default.chromium.png|11670| Changed|
|Field.Switch.default.chromium.png|9563| Changed|
|Field.Textarea.default.chromium.png|13275| Changed|
|Field.base.default.chromium.png|7992| Changed|
|Field.hint.default.chromium.png|8560| Changed|
|Field.horizontal+longLabel.default.chromium.png|9095| Changed|
|Field.horizontal+noLabel.default.chromium.png|8393| Changed|
|Field.horizontal.default.chromium.png|7557| Changed|
|Field.required.default.chromium.png|7992| Changed|
|Field.size-large.default.chromium.png|8159| Changed|
|Field.size-small.default.chromium.png|7791| Changed|
|Field.validation-error.default.chromium.png|8293| Changed|
|Field.validation-none.default.chromium.png|8293| Changed|
|Field.validation-success.default.chromium.png|9095| Changed|
|Field.validation-warning.default.chromium.png|8293| Changed|
|Field.validationMessageIcon.default.chromium.png|8293| Changed|
|FluentProvider CustomStyleHooks.Button.chromium.png|56956| Changed|
|FluentProvider CustomStyleHooks.CompoundButton.chromium.png|105026| Changed|
|FluentProvider CustomStyleHooks.Default.chromium.png|31708| Changed|
|FluentProvider CustomStyleHooks.MenuButton.chromium.png|56448| Changed|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|56130| Changed|
|FluentProvider CustomStyleHooks.ToggleButton.chromium.png|56956| Changed|
|Image Converged.Default.normal.chromium.png|410211| Changed|
|Image Converged.Image Fluid.normal.chromium.png|7958| Changed|
|Image Converged.Image Layout Fit.normal.chromium.png|1324312| Changed|
|Image Converged.Image Shadow.normal.chromium.png|49766| Changed|
|Image Converged.Image Shape.normal.chromium.png|885752| Changed|
|Image Converged.Image Variations Border.normal.chromium.png|886755| Changed|
|Image Converged.Image Variations Fallback.normal.chromium.png|238249| Changed|
|InfoButton.default - Dark Mode.active.chromium.png|7019| Changed|
|InfoButton.default - Dark Mode.focus.chromium.png|7022| Changed|
|InfoButton.default - Dark Mode.hover.chromium.png|7022| Changed|
|InfoButton.default - Dark Mode.rest.chromium.png|7022| Changed|
|InfoButton.default - High Contrast.active.chromium.png|7019| Changed|
|InfoButton.default - High Contrast.focus.chromium.png|7022| Changed|
|InfoButton.default - High Contrast.hover.chromium.png|7022| Changed|
|InfoButton.default - High Contrast.rest.chromium.png|7022| Changed|
|InfoButton.default.active.chromium.png|7021| Changed|
|InfoButton.default.focus.chromium.png|7022| Changed|
|InfoButton.default.hover.chromium.png|7022| Changed|
|InfoButton.default.rest.chromium.png|7022| Changed|
|InfoButton.size.rest.chromium.png|10327| Changed|
|InfoLabel.default - Dark Mode.rest.chromium.png|5684| Changed|
|InfoLabel.default - High Contrast.rest.chromium.png|5684| Changed|
|InfoLabel.default - RTL.rest.chromium.png|5684| Changed|
|InfoLabel.default.rest.chromium.png|5684| Changed|
|InfoLabel.required - RTL.rest.chromium.png|5684| Changed|
|InfoLabel.required.rest.chromium.png|5684| Changed|
|InfoLabel.size-large.rest.chromium.png|5718| Changed|
|InfoLabel.size-small.rest.chromium.png|5684| Changed|
|InfoLabel.wrap.rest.chromium.png|6353| Changed|
|Input Converged.Appearance- filled-darker.default.chromium.png|5885| Changed|
|Input Converged.Appearance- filled-darker.focused.chromium.png|5885| Changed|
|Input Converged.Appearance- filled-darker.hover.chromium.png|5885| Changed|
|Input Converged.Appearance- filled-lighter.default.chromium.png|6219| Changed|
|Input Converged.Appearance- filled-lighter.focused.chromium.png|6219| Changed|
|Input Converged.Appearance- filled-lighter.hover.chromium.png|6219| Changed|
|Input Converged.Appearance- outline (default).default.chromium.png|5885| Changed|
|Input Converged.Appearance- outline (default).focused.chromium.png|5885| Changed|
|Input Converged.Appearance- outline (default).hover.chromium.png|5885| Changed|
|Input Converged.Appearance- underline.default.chromium.png|5885| Changed|
|Input Converged.Appearance- underline.focused.chromium.png|5885| Changed|
|Input Converged.Appearance- underline.hover.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- filled-darker.default.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- filled-darker.focused.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- filled-darker.hover.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- filled-lighter.default.chromium.png|6219| Changed|
|Input Converged.Disabled, appearance- filled-lighter.focused.chromium.png|6219| Changed|
|Input Converged.Disabled, appearance- filled-lighter.hover.chromium.png|6219| Changed|
|Input Converged.Disabled, appearance- outline.default.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- outline.focused.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- outline.hover.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- underline.default.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- underline.focused.chromium.png|5885| Changed|
|Input Converged.Disabled, appearance- underline.hover.chromium.png|5885| Changed|
|Input Converged.Inline.default.chromium.png|13375| Changed|
|Input Converged.Invalid, appearance- filled-darker.default.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- filled-darker.focused.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- filled-darker.hover.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- filled-lighter.default.chromium.png|6219| Changed|
|Input Converged.Invalid, appearance- filled-lighter.focused.chromium.png|6219| Changed|
|Input Converged.Invalid, appearance- filled-lighter.hover.chromium.png|6219| Changed|
|Input Converged.Invalid, appearance- outline.default.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- outline.focused.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- outline.hover.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- underline.default.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- underline.focused.chromium.png|5885| Changed|
|Input Converged.Invalid, appearance- underline.hover.chromium.png|5885| Changed|
|Input Converged.Size- large.default.chromium.png|6019| Changed|
|Input Converged.Size- small.default.chromium.png|5751| Changed|
|Input Converged.With appearance override.default.chromium.png|6587| Changed|
|Input Converged.With value.default.chromium.png|5885| Changed|
|Input Converged.With value.focused.chromium.png|5885| Changed|
|Input Converged.With value.hover.chromium.png|5885| Changed|
|Input Converged.contentAfter - RTL.default.chromium.png|5885| Changed|
|Input Converged.contentAfter.default.chromium.png|5885| Changed|
|Input Converged.contentBefore - RTL.default.chromium.png|5885| Changed|
|Input Converged.contentBefore.default.chromium.png|5885| Changed|
|Menu.Nested Submenus Small Viewport Flipped.default.chromium.png|671356| Changed|
|Menu.Nested Submenus Small Viewport Flipped.nested menu.chromium.png|670274| Changed|
|Menu.Nested Submenus Small Viewport Stacked.default.chromium.png|433314| Changed|
|Menu.Nested Submenus Small Viewport Stacked.nested menu.chromium.png|432888| Changed|
|MenuButton Converged.Circular and icon only.default.chromium.png|50882| Changed|
|MenuButton Converged.Circular and icon only.hover.chromium.png|50817| Changed|
|MenuButton Converged.Circular and icon only.pressed.chromium.png|50832| Changed|
|MenuButton Converged.Circular.default.chromium.png|50686| Changed|
|MenuButton Converged.Circular.hover.chromium.png|50316| Changed|
|MenuButton Converged.Circular.pressed.chromium.png|50408| Changed|
|MenuButton Converged.Default.default.chromium.png|50674| Changed|
|MenuButton Converged.Default.hover.chromium.png|50278| Changed|
|MenuButton Converged.Default.pressed.chromium.png|50377| Changed|
|MenuButton Converged.Disabled.default.chromium.png|50793| Changed|
|MenuButton Converged.Disabled.hover.chromium.png|50793| Changed|
|MenuButton Converged.Disabled.pressed.chromium.png|50793| Changed|
|MenuButton Converged.Icon only.default.chromium.png|50869| Changed|
|MenuButton Converged.Icon only.hover.chromium.png|50780| Changed|
|MenuButton Converged.Icon only.pressed.chromium.png|50800| Changed|
|MenuButton Converged.Outline Disabled.default.chromium.png|50717| Changed|
|MenuButton Converged.Outline Disabled.hover.chromium.png|50717| Changed|
|MenuButton Converged.Outline Disabled.pressed.chromium.png|50717| Changed|
|MenuButton Converged.Outline.default.chromium.png|50674| Changed|
|MenuButton Converged.Outline.hover.chromium.png|50645| Changed|
|MenuButton Converged.Outline.pressed.chromium.png|50634| Changed|
|MenuButton Converged.Primary Disabled.default.chromium.png|50864| Changed|
|MenuButton Converged.Primary Disabled.hover.chromium.png|50864| Changed|
|MenuButton Converged.Primary Disabled.pressed.chromium.png|50864| Changed|
|MenuButton Converged.Primary.default.chromium.png|47848| Changed|
|MenuButton Converged.Primary.hover.chromium.png|47847| Changed|
|MenuButton Converged.Primary.pressed.chromium.png|47705| Changed|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|129356| Changed|
|MenuButton Converged.Size large.default.chromium.png|63236| Changed|
|MenuButton Converged.Size large.hover.chromium.png|62553| Changed|
|MenuButton Converged.Size large.pressed.chromium.png|62736| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|81784| Changed|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|56458| Changed|
|MenuButton Converged.Size small.default.chromium.png|37955| Changed|
|MenuButton Converged.Size small.hover.chromium.png|37676| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|37742| Changed|
|MenuButton Converged.Subtle Disabled.default.chromium.png|50781| Changed|
|MenuButton Converged.Subtle Disabled.hover.chromium.png|50781| Changed|
|MenuButton Converged.Subtle Disabled.pressed.chromium.png|50781| Changed|
|MenuButton Converged.Subtle.default.chromium.png|50730| Changed|
|MenuButton Converged.Subtle.hover.chromium.png|50462| Changed|
|MenuButton Converged.Subtle.pressed.chromium.png|50314| Changed|
|MenuButton Converged.Transparent Disabled.default.chromium.png|50781| Changed|
|MenuButton Converged.Transparent Disabled.hover.chromium.png|50781| Changed|
|MenuButton Converged.Transparent Disabled.pressed.chromium.png|50781| Changed|
|MenuButton Converged.Transparent.default.chromium.png|50730| Changed|
|MenuButton Converged.Transparent.hover.chromium.png|50665| Changed|
|MenuButton Converged.Transparent.pressed.chromium.png|50659| Changed|
|MenuButton Converged.With icon.default.chromium.png|50638| Changed|
|MenuButton Converged.With icon.hover.chromium.png|50172| Changed|
|MenuButton Converged.With icon.pressed.chromium.png|50290| Changed|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|60712| Changed|
|Popover Converged.avoid scrolling.chromium.png|63448| Changed|
|Popover Converged.positioning - High Contrast.chromium.png|536241| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|20464| Changed|
|Popover Converged.positioning - RTL.chromium.png|536241| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|20464| Changed|
|Popover Converged.positioning.chromium.png|536241| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|20464| Changed|
|Portal.Apply Class Names.chromium.png|32816| Changed|
|Portal.Apply Class Names.should have green border.chromium.png|40686| Changed|
|PresenceBadge Converged - inverted background.default - Dark Mode.default.chromium.png|12506| Changed|
|PresenceBadge Converged - inverted background.default - High Contrast.default.chromium.png|12506| Changed|
|PresenceBadge Converged - inverted background.default.default.chromium.png|12506| Changed|
|Radio Converged.checked - Dark Mode.active.chromium.png|5885| Changed|
|Radio Converged.checked - Dark Mode.hover.chromium.png|5885| Changed|
|Radio Converged.checked - Dark Mode.rest.chromium.png|5885| Changed|
|Radio Converged.checked - High Contrast.active.chromium.png|5885| Changed|
|Radio Converged.checked - High Contrast.hover.chromium.png|5885| Changed|
|Radio Converged.checked - High Contrast.rest.chromium.png|5885| Changed|
|Radio Converged.checked.active.chromium.png|5885| Changed|
|Radio Converged.checked.hover.chromium.png|5885| Changed|
|Radio Converged.checked.rest.chromium.png|5885| Changed|
|Radio Converged.disabled - Dark Mode.active.chromium.png|5885| Changed|
|Radio Converged.disabled - Dark Mode.hover.chromium.png|5885| Changed|
|Radio Converged.disabled - Dark Mode.rest.chromium.png|5885| Changed|
|Radio Converged.disabled - High Contrast.active.chromium.png|5885| Changed|
|Radio Converged.disabled - High Contrast.hover.chromium.png|5885| Changed|
|Radio Converged.disabled - High Contrast.rest.chromium.png|5885| Changed|
|Radio Converged.disabled+checked - Dark Mode.default.chromium.png|5885| Changed|
|Radio Converged.disabled+checked - High Contrast.default.chromium.png|5885| Changed|
|Radio Converged.disabled+checked.default.chromium.png|5885| Changed|
|Radio Converged.disabled.active.chromium.png|5885| Changed|
|Radio Converged.disabled.hover.chromium.png|5885| Changed|
|Radio Converged.disabled.rest.chromium.png|5885| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|6955| Changed|
|Radio Converged.group-disabled.default.chromium.png|6955| Changed|
|Radio Converged.group-horizontal - RTL.default.chromium.png|5885| Changed|
|Radio Converged.group-horizontal-stacked - RTL.default.chromium.png|6420| Changed|
|Radio Converged.group-horizontal-stacked.default.chromium.png|6420| Changed|
|Radio Converged.group-horizontal.default.chromium.png|5885| Changed|
|Radio Converged.group-vertical - RTL.default.chromium.png|6955| Changed|
|Radio Converged.group-vertical.default.chromium.png|6955| Changed|
|Radio Converged.label-after - RTL.default.chromium.png|5885| Changed|
|Radio Converged.label-after.default.chromium.png|5885| Changed|
|Radio Converged.label-below - RTL.default.chromium.png|6420| Changed|
|Radio Converged.label-below.default.chromium.png|6420| Changed|
|Radio Converged.label-wrapping - RTL.default.chromium.png|6888| Changed|
|Radio Converged.label-wrapping.default.chromium.png|6888| Changed|
|Radio Converged.no-label - RTL.default.chromium.png|5885| Changed|
|Radio Converged.no-label.default.chromium.png|5885| Changed|
|Radio Converged.required - RTL.default.chromium.png|5885| Changed|
|Radio Converged.required.default.chromium.png|5885| Changed|
|Radio Converged.unchecked - Dark Mode.active.chromium.png|5885| Changed|
|Radio Converged.unchecked - Dark Mode.hover.chromium.png|5885| Changed|
|Radio Converged.unchecked - Dark Mode.rest.chromium.png|5885| Changed|
|Radio Converged.unchecked - High Contrast.active.chromium.png|5885| Changed|
|Radio Converged.unchecked - High Contrast.hover.chromium.png|5885| Changed|
|Radio Converged.unchecked - High Contrast.rest.chromium.png|5885| Changed|
|Radio Converged.unchecked.active.chromium.png|5885| Changed|
|Radio Converged.unchecked.hover.chromium.png|5885| Changed|
|Radio Converged.unchecked.rest.chromium.png|5885| Changed|
|Select Converged.Appearance- filled-darker.default.chromium.png|6219| Changed|
|Select Converged.Appearance- filled-darker.focused.chromium.png|6219| Changed|
|Select Converged.Appearance- filled-darker.hover.chromium.png|6219| Changed|
|Select Converged.Appearance- filled-lighter.default.chromium.png|6219| Changed|
|Select Converged.Appearance- filled-lighter.focused.chromium.png|6219| Changed|
|Select Converged.Appearance- filled-lighter.hover.chromium.png|6219| Changed|
|Select Converged.Appearance- outline (default).default.chromium.png|5885| Changed|
|Select Converged.Appearance- outline (default).focused.chromium.png|5885| Changed|
|Select Converged.Appearance- outline (default).hover.chromium.png|5885| Changed|
|Select Converged.Appearance- underline.default.chromium.png|5885| Changed|
|Select Converged.Appearance- underline.focused.chromium.png|5885| Changed|
|Select Converged.Appearance- underline.hover.chromium.png|5885| Changed|
|Select Converged.Custom Icon.default.chromium.png|5885| Changed|
|Select Converged.Disabled.default.chromium.png|5885| Changed|
|Select Converged.Invalid- filled-darker.default.chromium.png|6219| Changed|
|Select Converged.Invalid- filled-darker.focused.chromium.png|6219| Changed|
|Select Converged.Invalid- filled-darker.hover.chromium.png|6219| Changed|
|Select Converged.Invalid- filled-lighter.default.chromium.png|6219| Changed|
|Select Converged.Invalid- filled-lighter.focused.chromium.png|6219| Changed|
|Select Converged.Invalid- filled-lighter.hover.chromium.png|6219| Changed|
|Select Converged.Invalid- outline.default.chromium.png|5885| Changed|
|Select Converged.Invalid- outline.focused.chromium.png|5885| Changed|
|Select Converged.Invalid- outline.hover.chromium.png|5885| Changed|
|Select Converged.Invalid- underline.default.chromium.png|5885| Changed|
|Select Converged.Invalid- underline.focused.chromium.png|5885| Changed|
|Select Converged.Invalid- underline.hover.chromium.png|5885| Changed|
|Select Converged.Size- large.default.chromium.png|6019| Changed|
|Select Converged.Size- small.default.chromium.png|5751| Changed|
|Select Converged.With appearance override.default.chromium.png|6587| Changed|
|Select Converged.With value.default.chromium.png|5885| Changed|
|Skeleton converged.Opaque Skeleton with circle - Dark Mode.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with circle - High Contrast.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with circle - RTL.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with circle.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - Dark Mode.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - High Contrast.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - RTL.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with rectangle.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with square - Dark Mode.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with square - High Contrast.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with square - RTL.chromium.png|32100| Changed|
|Skeleton converged.Opaque Skeleton with square.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with  circle - Dark Mode.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with  circle - High Contrast.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with  circle - RTL.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with  circle.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - Dark Mode.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - High Contrast.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - RTL.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with rectangle.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with square - Dark Mode.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with square - High Contrast.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with square - RTL.chromium.png|32100| Changed|
|Skeleton converged.Translucent Skeleton with square.chromium.png|32100| Changed|
|Slider Converged.Disabled - Dark Mode.default.chromium.png|3026| Changed|
|Slider Converged.Disabled - Dark Mode.hover.chromium.png|3026| Changed|
|Slider Converged.Disabled - Dark Mode.pressed.chromium.png|3026| Changed|
|Slider Converged.Disabled - High Contrast.default.chromium.png|3026| Changed|
|Slider Converged.Disabled - High Contrast.hover.chromium.png|3026| Changed|
|Slider Converged.Disabled - High Contrast.pressed.chromium.png|3026| Changed|
|Slider Converged.Disabled Vertical.default.chromium.png|2909| Changed|
|Slider Converged.Disabled Vertical.hover.chromium.png|2909| Changed|
|Slider Converged.Disabled Vertical.pressed.chromium.png|2909| Changed|
|Slider Converged.Disabled.default.chromium.png|3026| Changed|
|Slider Converged.Disabled.hover.chromium.png|3026| Changed|
|Slider Converged.Disabled.pressed.chromium.png|3026| Changed|
|Slider Converged.Horizontal - 0% - RTL.default.chromium.png|3026| Changed|
|Slider Converged.Horizontal - 0%.default.chromium.png|3026| Changed|
|Slider Converged.Horizontal - 100% - RTL.default.chromium.png|3026| Changed|
|Slider Converged.Horizontal - 100%.default.chromium.png|3026| Changed|
|Slider Converged.Root - Dark Mode.default.chromium.png|3026| Changed|
|Slider Converged.Root - Dark Mode.hover.chromium.png|3026| Changed|
|Slider Converged.Root - Dark Mode.pressed.chromium.png|3026| Changed|
|Slider Converged.Root - High Contrast.default.chromium.png|3026| Changed|
|Slider Converged.Root - High Contrast.hover.chromium.png|3026| Changed|
|Slider Converged.Root - High Contrast.pressed.chromium.png|3026| Changed|
|Slider Converged.Root - RTL.default.chromium.png|3026| Changed|
|Slider Converged.Root - RTL.hover.chromium.png|3026| Changed|
|Slider Converged.Root - RTL.pressed.chromium.png|3026| Changed|
|Slider Converged.Root.default.chromium.png|3026| Changed|
|Slider Converged.Root.hover.chromium.png|3026| Changed|
|Slider Converged.Root.pressed.chromium.png|3026| Changed|
|Slider Converged.Vertical - 0% - RTL.default.chromium.png|2909| Changed|
|Slider Converged.Vertical - 0%.default.chromium.png|2909| Changed|
|Slider Converged.Vertical - 100% - RTL.default.chromium.png|2909| Changed|
|Slider Converged.Vertical - 100%.default.chromium.png|2909| Changed|
|Slider Converged.Vertical - RTL.default.chromium.png|2909| Changed|
|Slider Converged.Vertical - RTL.hover.chromium.png|2909| Changed|
|Slider Converged.Vertical - RTL.pressed.chromium.png|2909| Changed|
|Slider Converged.Vertical.default.chromium.png|2909| Changed|
|Slider Converged.Vertical.hover.chromium.png|2909| Changed|
|Slider Converged.Vertical.pressed.chromium.png|2909| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.focused.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverInput.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.rest.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter.focused.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverInput.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter.rest.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default).focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default).hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default).hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default).hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default).rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - RTL.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - RTL.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - RTL.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - RTL.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - RTL.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound.rest.chromium.png|5885| Changed|
|SpinButton Converged.Custom Width - Dark Mode.chromium.png|74740| Changed|
|SpinButton Converged.Custom Width - High Contrast.chromium.png|74740| Changed|
|SpinButton Converged.Custom Width - RTL.chromium.png|74740| Changed|
|SpinButton Converged.Custom Width.chromium.png|74740| Changed|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|5885| Changed|
|SpinButton Converged.Disabled.focused.chromium.png|5885| Changed|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled.rest.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - RTL.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - RTL.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - RTL.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - RTL.rest.chromium.png|5885| Changed|
|SpinButton Converged.Display Value.focused.chromium.png|5885| Changed|
|SpinButton Converged.Display Value.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value.rest.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-darker.rest.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|6219| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverInput.chromium.png|6219| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Invalid- filled-lighter.rest.chromium.png|6219| Changed|
|SpinButton Converged.Invalid- outline.focused.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- outline.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- outline.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- outline.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- outline.rest.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- underline.focused.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- underline.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- underline.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- underline.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Invalid- underline.rest.chromium.png|5885| Changed|
|SpinButton Converged.Size- medium (default) - RTL.default.chromium.png|5885| Changed|
|SpinButton Converged.Size- medium (default).default.chromium.png|5885| Changed|
|SpinButton Converged.Size- small - Dark Mode.default.chromium.png|5751| Changed|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|5751| Changed|
|SpinButton Converged.Size- small - RTL.default.chromium.png|5751| Changed|
|SpinButton Converged.Size- small.default.chromium.png|5751| Changed|
|SpinButton Converged.With appearance override.default.chromium.png|6587| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|112337| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|112337| Changed|
|Spinner converged.Inverted with Huge - Dark Mode.chromium.png|130273| Changed|
|Spinner converged.Inverted with Huge - High Contrast.chromium.png|130273| Changed|
|Spinner converged.Inverted with Huge.chromium.png|130273| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label Above - Dark Mode.chromium.png|156725| Changed|
|Spinner converged.Inverted with Label Above - High Contrast.chromium.png|156725| Changed|
|Spinner converged.Inverted with Label Above.chromium.png|156725| Changed|
|Spinner converged.Inverted with Label After - Dark Mode.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label After - High Contrast.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label After.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label Before - Dark Mode.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label Before - High Contrast.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label Before.chromium.png|109294| Changed|
|Spinner converged.Inverted with Label Below - Dark Mode.chromium.png|156725| Changed|
|Spinner converged.Inverted with Label Below - High Contrast.chromium.png|156725| Changed|
|Spinner converged.Inverted with Label Below.chromium.png|156725| Changed|
|Spinner converged.Inverted with Label.chromium.png|109294| Changed|
|Spinner converged.Inverted with Size Extra Large - Dark Mode.chromium.png|124321| Changed|
|Spinner converged.Inverted with Size Extra Large - High Contrast.chromium.png|124321| Changed|
|Spinner converged.Inverted with Size Extra Large.chromium.png|124321| Changed|
|Spinner converged.Inverted with Size Extra Small - Dark Mode.chromium.png|100246| Changed|
|Spinner converged.Inverted with Size Extra Small - High Contrast.chromium.png|100246| Changed|
|Spinner converged.Inverted with Size Extra Small.chromium.png|100246| Changed|
|Spinner converged.Inverted with Size Large - Dark Mode.chromium.png|118342| Changed|
|Spinner converged.Inverted with Size Large - High Contrast.chromium.png|118342| Changed|
|Spinner converged.Inverted with Size Large.chromium.png|118342| Changed|
|Spinner converged.Inverted with Size Medium - Dark Mode.chromium.png|112337| Changed|
|Spinner converged.Inverted with Size Medium - High Contrast.chromium.png|112337| Changed|
|Spinner converged.Inverted with Size Medium.chromium.png|112337| Changed|
|Spinner converged.Inverted with Size Small - Dark Mode.chromium.png|106305| Changed|
|Spinner converged.Inverted with Size Small - High Contrast.chromium.png|106305| Changed|
|Spinner converged.Inverted with Size Small.chromium.png|106305| Changed|
|Spinner converged.Inverted.chromium.png|112337| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|81909| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|81909| Changed|
|Spinner converged.Primary with Huge - Dark Mode.chromium.png|100246| Changed|
|Spinner converged.Primary with Huge - High Contrast.chromium.png|100246| Changed|
|Spinner converged.Primary with Huge.chromium.png|100246| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|80036| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|80036| Changed|
|Spinner converged.Primary with Label Above - Dark Mode.chromium.png|127300| Changed|
|Spinner converged.Primary with Label Above - High Contrast.chromium.png|127300| Changed|
|Spinner converged.Primary with Label Above.chromium.png|127300| Changed|
|Spinner converged.Primary with Label After - Dark Mode.chromium.png|80036| Changed|
|Spinner converged.Primary with Label After - High Contrast.chromium.png|80036| Changed|
|Spinner converged.Primary with Label After - RTL.chromium.png|80036| Changed|
|Spinner converged.Primary with Label After.chromium.png|80036| Changed|
|Spinner converged.Primary with Label Before - Dark Mode.chromium.png|80036| Changed|
|Spinner converged.Primary with Label Before - High Contrast.chromium.png|80036| Changed|
|Spinner converged.Primary with Label Before - RTL.chromium.png|80036| Changed|
|Spinner converged.Primary with Label Before.chromium.png|80036| Changed|
|Spinner converged.Primary with Label Below - Dark Mode.chromium.png|127300| Changed|
|Spinner converged.Primary with Label Below - High Contrast.chromium.png|127300| Changed|
|Spinner converged.Primary with Label Below.chromium.png|127300| Changed|
|Spinner converged.Primary with Label.chromium.png|80036| Changed|
|Spinner converged.Primary with Size Extra Large - Dark Mode.chromium.png|94160| Changed|
|Spinner converged.Primary with Size Extra Large - High Contrast.chromium.png|94160| Changed|
|Spinner converged.Primary with Size Extra Large.chromium.png|94160| Changed|
|Spinner converged.Primary with Size Extra Small - Dark Mode.chromium.png|69550| Changed|
|Spinner converged.Primary with Size Extra Small - High Contrast.chromium.png|69550| Changed|
|Spinner converged.Primary with Size Extra Small.chromium.png|69550| Changed|
|Spinner converged.Primary with Size Large - Dark Mode.chromium.png|88048| Changed|
|Spinner converged.Primary with Size Large - High Contrast.chromium.png|88048| Changed|
|Spinner converged.Primary with Size Large.chromium.png|88048| Changed|
|Spinner converged.Primary with Size Medium - Dark Mode.chromium.png|81909| Changed|
|Spinner converged.Primary with Size Medium - High Contrast.chromium.png|81909| Changed|
|Spinner converged.Primary with Size Medium.chromium.png|81909| Changed|
|Spinner converged.Primary with Size Small - Dark Mode.chromium.png|75743| Changed|
|Spinner converged.Primary with Size Small - High Contrast.chromium.png|75743| Changed|
|Spinner converged.Primary with Size Small.chromium.png|75743| Changed|
|Spinner converged.Primary with Size Tiny - Dark Mode.chromium.png|94160| Changed|
|Spinner converged.Primary with Size Tiny - High Contrast.chromium.png|94160| Changed|
|Spinner converged.Primary with Size Tiny.chromium.png|94160| Changed|
|Spinner converged.Primary.chromium.png|81909| Changed|
|Switch Converged.Disabled and checked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - High Contrast.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - High Contrast.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - High Contrast.pressed.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked.pressed.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.pressed.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - High Contrast.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - High Contrast.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - High Contrast.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - RTL.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - RTL.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - RTL.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - RTL.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - RTL.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - RTL.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked.pressed.chromium.png|3524| Changed|
|Switch Converged.Required.default.chromium.png|3645| Changed|
|Switch Converged.Required.hover.chromium.png|3645| Changed|
|Switch Converged.Required.pressed.chromium.png|3645| Changed|
|Switch Converged.With label above - RTL.default.chromium.png|3006| Changed|
|Switch Converged.With label above - RTL.hover.chromium.png|3006| Changed|
|Switch Converged.With label above - RTL.pressed.chromium.png|3006| Changed|
|Switch Converged.With label above.default.chromium.png|3006| Changed|
|Switch Converged.With label above.hover.chromium.png|3006| Changed|
|Switch Converged.With label above.pressed.chromium.png|3006| Changed|
|Switch Converged.With label after - RTL.default.chromium.png|3524| Changed|
|Switch Converged.With label after - RTL.hover.chromium.png|3524| Changed|
|Switch Converged.With label after - RTL.pressed.chromium.png|3524| Changed|
|Switch Converged.With label after.default.chromium.png|3524| Changed|
|Switch Converged.With label after.hover.chromium.png|3524| Changed|
|Switch Converged.With label after.pressed.chromium.png|3524| Changed|
|Switch Converged.With label before - RTL.default.chromium.png|3524| Changed|
|Switch Converged.With label before - RTL.hover.chromium.png|3524| Changed|
|Switch Converged.With label before - RTL.pressed.chromium.png|3524| Changed|
|Switch Converged.With label before.default.chromium.png|3524| Changed|
|Switch Converged.With label before.hover.chromium.png|3524| Changed|
|Switch Converged.With label before.pressed.chromium.png|3524| Changed|
|Switch Converged.With label wrapping.default.chromium.png|14763| Changed|
|Switch Converged.With label wrapping.hover.chromium.png|14763| Changed|
|Switch Converged.With label wrapping.pressed.chromium.png|14763| Changed|
|Textarea Converged.Appearance- filled-darker.default.chromium.png|6286| Changed|
|Textarea Converged.Appearance- filled-darker.focused.chromium.png|106920| Changed|
|Textarea Converged.Appearance- filled-darker.hover.chromium.png|6286| Changed|
|Textarea Converged.Appearance- filled-lighter.default.chromium.png|6621| Changed|
|Textarea Converged.Appearance- filled-lighter.focused.chromium.png|133737| Changed|
|Textarea Converged.Appearance- filled-lighter.hover.chromium.png|6621| Changed|
|Textarea Converged.Appearance- outline (default).default.chromium.png|6286| Changed|
|Textarea Converged.Appearance- outline (default).focused.chromium.png|106920| Changed|
|Textarea Converged.Appearance- outline (default).hover.chromium.png|6286| Changed|
|Textarea Converged.Disabled.default.chromium.png|6286| Changed|
|Textarea Converged.Disabled.focused.chromium.png|106920| Changed|
|Textarea Converged.Disabled.hover.chromium.png|6286| Changed|
|Textarea Converged.Invalid- filled-darker.default.chromium.png|6286| Changed|
|Textarea Converged.Invalid- filled-darker.focused.chromium.png|106920| Changed|
|Textarea Converged.Invalid- filled-darker.hover.chromium.png|6286| Changed|
|Textarea Converged.Invalid- filled-lighter.default.chromium.png|6621| Changed|
|Textarea Converged.Invalid- filled-lighter.focused.chromium.png|133737| Changed|
|Textarea Converged.Invalid- filled-lighter.hover.chromium.png|6621| Changed|
|Textarea Converged.Invalid- outline.default.chromium.png|6286| Changed|
|Textarea Converged.Invalid- outline.focused.chromium.png|106920| Changed|
|Textarea Converged.Invalid- outline.hover.chromium.png|6286| Changed|
|Textarea Converged.Size- large.default.chromium.png|6487| Changed|
|Textarea Converged.Size- medium.default.chromium.png|6286| Changed|
|Textarea Converged.Size- small.default.chromium.png|6086| Changed|
|Textarea Converged.With appearance override.default.chromium.png|7390| Changed|
|Textarea Converged.With value.default.chromium.png|6286| Changed|
|Textarea Converged.With value.focused.chromium.png|106920| Changed|
|Textarea Converged.With value.hover.chromium.png|6286| Changed|
|ToggleButton Converged.Checked.default.chromium.png|50330| Changed|
|ToggleButton Converged.Checked.hover.chromium.png|50356| Changed|
|ToggleButton Converged.Checked.pressed.chromium.png|50439| Changed|
|ToggleButton Converged.Circular and icon only.default.chromium.png|50882| Changed|
|ToggleButton Converged.Circular and icon only.hover.chromium.png|50817| Changed|
|ToggleButton Converged.Circular and icon only.pressed.chromium.png|50832| Changed|
|ToggleButton Converged.Circular.default.chromium.png|50706| Changed|
|ToggleButton Converged.Circular.hover.chromium.png|50393| Changed|
|ToggleButton Converged.Circular.pressed.chromium.png|50471| Changed|
|ToggleButton Converged.Default.default.chromium.png|50694| Changed|
|ToggleButton Converged.Default.hover.chromium.png|50356| Changed|
|ToggleButton Converged.Default.pressed.chromium.png|50439| Changed|
|ToggleButton Converged.Disabled.default.chromium.png|50792| Changed|
|ToggleButton Converged.Disabled.hover.chromium.png|50792| Changed|
|ToggleButton Converged.Disabled.pressed.chromium.png|50792| Changed|
|ToggleButton Converged.Icon only.default.chromium.png|50869| Changed|
|ToggleButton Converged.Icon only.hover.chromium.png|50780| Changed|
|ToggleButton Converged.Icon only.pressed.chromium.png|50800| Changed|
|ToggleButton Converged.Outline.default.chromium.png|50694| Changed|
|ToggleButton Converged.Outline.hover.chromium.png|50668| Changed|
|ToggleButton Converged.Outline.pressed.chromium.png|50658| Changed|
|ToggleButton Converged.Primary Checked.default.chromium.png|48133| Changed|
|ToggleButton Converged.Primary Checked.hover.chromium.png|48254| Changed|
|ToggleButton Converged.Primary Checked.pressed.chromium.png|48133| Changed|
|ToggleButton Converged.Primary Disabled.default.chromium.png|50856| Changed|
|ToggleButton Converged.Primary Disabled.hover.chromium.png|50856| Changed|
|ToggleButton Converged.Primary Disabled.pressed.chromium.png|50856| Changed|
|ToggleButton Converged.Primary.default.chromium.png|48255| Changed|
|ToggleButton Converged.Primary.hover.chromium.png|48254| Changed|
|ToggleButton Converged.Primary.pressed.chromium.png|48133| Changed|
|ToggleButton Converged.Size large - with long text wrapping.chromium.png|129380| Changed|
|ToggleButton Converged.Size large.default.chromium.png|63262| Changed|
|ToggleButton Converged.Size large.hover.chromium.png|62670| Changed|
|ToggleButton Converged.Size large.pressed.chromium.png|62828| Changed|
|ToggleButton Converged.Size medium - with long text wrapping.chromium.png|81790| Changed|
|ToggleButton Converged.Size small - with long text wrapping.chromium.png|56533| Changed|
|ToggleButton Converged.Size small.default.chromium.png|37975| Changed|
|ToggleButton Converged.Size small.hover.chromium.png|37739| Changed|
|ToggleButton Converged.Size small.pressed.chromium.png|37794| Changed|
|ToggleButton Converged.Subtle Checked.default.chromium.png|50460| Changed|
|ToggleButton Converged.Subtle Checked.hover.chromium.png|50517| Changed|
|ToggleButton Converged.Subtle Checked.pressed.chromium.png|50392| Changed|
|ToggleButton Converged.Subtle Disabled.default.chromium.png|50785| Changed|
|ToggleButton Converged.Subtle Disabled.hover.chromium.png|50785| Changed|
|ToggleButton Converged.Subtle Disabled.pressed.chromium.png|50785| Changed|
|ToggleButton Converged.Subtle.default.chromium.png|50737| Changed|
|ToggleButton Converged.Subtle.hover.chromium.png|50517| Changed|
|ToggleButton Converged.Subtle.pressed.chromium.png|50392| Changed|
|ToggleButton Converged.Transparent Checked.default.chromium.png|50672| Changed|
|ToggleButton Converged.Transparent Checked.hover.chromium.png|50672| Changed|
|ToggleButton Converged.Transparent Checked.pressed.chromium.png|50666| Changed|
|ToggleButton Converged.Transparent Disabled.default.chromium.png|50785| Changed|
|ToggleButton Converged.Transparent Disabled.hover.chromium.png|50785| Changed|
|ToggleButton Converged.Transparent Disabled.pressed.chromium.png|50785| Changed|
|ToggleButton Converged.Transparent.default.chromium.png|50737| Changed|
|ToggleButton Converged.Transparent.hover.chromium.png|50672| Changed|
|ToggleButton Converged.Transparent.pressed.chromium.png|50666| Changed|
|ToggleButton Converged.With icon after content.default.chromium.png|50658| Changed|
|ToggleButton Converged.With icon after content.hover.chromium.png|50250| Changed|
|ToggleButton Converged.With icon after content.pressed.chromium.png|50352| Changed|
|ToggleButton Converged.With icon before content.default.chromium.png|50658| Changed|
|ToggleButton Converged.With icon before content.hover.chromium.png|50250| Changed|
|ToggleButton Converged.With icon before content.pressed.chromium.png|50352| Changed|
|Toolbar Converged.Default.Button Pressed.chromium.png|22933| Changed|
|Toolbar Converged.Default.Toggle On.chromium.png|23000| Changed|
|Toolbar Converged.Default.default.chromium.png|23103| Changed|
|Toolbar Converged.Far Group.chromium.png|72901| Changed|
|Toolbar Converged.Large.Button Pressed.chromium.png|26946| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|27013| Changed|
|Toolbar Converged.Large.default.chromium.png|27115| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|18921| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|18988| Changed|
|Toolbar Converged.Small.default.chromium.png|19090| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|19209| Changed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|18849| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|19247| Changed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|83619| Changed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|83686| Changed|
|Toolbar Converged.Vertical.default.chromium.png|83788| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|192288| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|192288| Changed|
|Tooltip Converged.basic.chromium.png|192288| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|192288| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|192288| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|192288| Changed|
|Tooltip Converged.inverted.chromium.png|192288| Changed|
|Tooltip Converged.positioning - High Contrast.chromium.png|489528| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|11823| Changed|
|Tooltip Converged.positioning - RTL.chromium.png|489528| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|11856| Changed|
|Tooltip Converged.positioning.chromium.png|489528| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|11856| Changed|
|Tooltip Converged.text-wrapping.chromium.png|192265| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|192288| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|192288| Changed|
|Tooltip Converged.withArrow.chromium.png|192288| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>